Adds German translation

bttrx opened the pull request.

albertvaka commented:
Thanks a lot for the translation! 😄 Sorry I messed up when merging this (I force-pushed my main branch and it caused a conflict here) and this shows as "closed" instead of "merged" 😓

bttrx commented:
> Thanks a lot for the translation! 😄 Sorry I messed up when merging this (I force-pushed my main branch and it caused a conflict here) and this shows as "closed" instead of "merged" 😓

Yes, the conflict caused me headache for my fork, because I'm a Git noob. Finally, I deleted my fork and created a new one. Also deleted my local clone (staging area / working directory).

albertvaka commented:
Sorry, that's totally my fault. One should never force-push master, but I didn't expect anyone to have cloned my repo so I thought it was safe 😅  Sorry again and thanks for your contribution!
